arm/Microsoft.EventHub/namespaces/deploy.bicep

```diff
@@ -230,13 +230,11 @@ module eventHubNamespace_eventHubs 'eventhubs/deploy.bicep' = [for (eventHub, in
     authorizationRules: contains(eventHub, 'authorizationRules') ? eventHub.authorizationRules : [
       {
         name: 'RootManageSharedAccessKey'
-        properties: {
-          rights: [
-            'Listen'
-            'Manage'
-            'Send'
-          ]
-        }
+        rights: [
```

Comment:
Are we testing this in the parameters.json?

Reply:
Good point. I don't think so. We'd need to add a test case with an empty event hub. Will give it a spin

Reply:
done